src/status_im2/contexts/wallet/account/tabs/view.cljs

```diff
@@ -6,22 +6,23 @@
     [status-im2.contexts.wallet.common.activity-tab.view :as activity]
     [status-im2.contexts.wallet.common.collectibles-tab.view :as collectibles]
     [status-im2.contexts.wallet.common.empty-tab.view :as empty-tab]
-    [status-im2.contexts.wallet.common.temp :as temp]
     [status-im2.contexts.wallet.common.token-value.view :as token-value]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
 
 (defn view
-  [{:keys [selected-tab]}]
-  (case selected-tab
-    :assets       [rn/flat-list
-                   {:render-fn               token-value/view
-                    :data                    temp/tokens
-                    :content-container-style {:padding-horizontal 8}}]
-    :collectibles [collectibles/view]
-    :activity     [activity/view]
-    :permissions  [empty-tab/view
-                   {:title        (i18n/label :t/no-permissions)
-                    :description  (i18n/label :t/no-collectibles-description)
-                    :placeholder? true}]
-    :dapps        [dapps/view]
-    [about/view]))
+  [{:keys [selected-tab account-address]}]
+  (let [parsed-tokens (rf/sub [:wallet/parsed-tokens account-address])]
```

Comment:
Will add a few naming nitpicks)
Within this view, the verb `parsed` doesn't add any understanding for the reader, and maybe even confuses a bit because unclear what was parsed and why. I'd leave just `tokens`